README.md

```diff
@@ -1,2 +1,30 @@
-# whisktool
-Utility for managing OpenWhisk Projects
+# Whisk Deploy `wskdeploy`
+
+`wskdeploy` is a utility to help you create and deploy OpenWhisk projects. Deploy all your actions, triggers, and rules using a single command! You can use this in addition to the OpenWhisk CLI.
+
+`wskdeploy` is currenty under development and in its very early stages.  Check back often to see how its progressing.
+
+# How to Build
+`wskdeploy` is written in Go.
+
+1. Setup your [Go development environment](https://golang.org/doc/code.html).
+
+2.  `wskdeploy` depends on the `github.com/openwhisk-client-go/whisk` . To install:
```

Comment:
`github.com/openwhisk-client-go/whisk` should be `github.com/openwhisk/openwhisk-client-go/whisk`